cmd/rancherd/bundle/share/rancherd/rke2-cis-sysctl.conf

```diff
@@ -0,0 +1,5 @@
+vm.panic_on_oom=0
```

Comment:
Should this file name be `rancherd-cis-sysctl`? not sure if you want to rename all these to rancherd

Reply:
It didn't seem necessary because `rancherd` is a very thin wrapper around `rke2` so I left it. This has the benefit of having the same file name that is mentioned in the rke2 readme (althought it is under share/rancherd instead of share/rke2). Happy to change if someone objects.